network-api/networkapi/templates/fragments/buyersguide/it_uses.html

```diff
@@ -3,7 +3,7 @@
 <div class="it-uses d-flex flex-column">
     <h4 class="pni-product-smaller-body-bold">{{ title }}</h4>
     <div class="explanation">
-        <p class="pni-product-smaller-body mb-0">{% trans "Device:" %} <span class="font-italic">{{ device|extended_yes_no }}</span></p>
-        <p class="pni-product-smaller-body mb-0">{% trans "App:" %} <span class="font-italic">{{ app|extended_yes_no }}</span></p>
+        <p class="pni-product-smaller-body mb-0">{% trans "Device:" %} <span class="font-italic">{{ device|extended_choice }}</span></p>
+        <p class="pni-product-smaller-body mb-0">{% trans "App:" %} <span class="font-italic">{{ app|extended_choice }}</span></p>
```

Comment:
While not directly related to the field `ai_is_untrustworthy`, these have been updated to use the updated template tag mentioned [here](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/12976/files#r1791075302)